ui/raidboss/timeline.js

```diff
@@ -43,11 +43,11 @@ class Timeline {
       return text;
 
     let orig = text;
-    let locale = this.options.Language || 'en';
+    let parserLang = this.options.PraserLanguage || 'en';
```

Comment:
typo, here and below

also maybe this variable should just be called "replaceLang" or just "lang" since it's not necessarily the parser language.